tests/providers/cncf/kubernetes/executors/test_kubernetes_executor.py

```diff
@@ -1247,6 +1250,11 @@ def test_clear_not_launched_queued_tasks_not_launched(
         mock_kube_dynamic_client.return_value.resources.get.return_value = mock_pod_resource
         mock_kube_dynamic_client.return_value.get.return_value.items = []
 
+        # This is hack to use overridden conf vars as it seems executors loaded before conf override.
```

Comment:
The ExecutorLoader holds state at the module level so that it's only loaded once and unique across instances. But that leads to weird stuff like this in tests. If you look at other examples we use strategic calls to `reload()` that module. I haven't verified if what you're doing here works just as well.

Reply:
yes clear works, but reload seems better, updated